packages/react-tabs/src/components/Tabs/Tabs.base.tsx

```diff
@@ -131,7 +131,7 @@ export const TabsBase: React.FunctionComponent<TabsProps> = React.forwardRef<HTM
           // eslint-disable-next-line react/jsx-no-bind
           onKeyDown={(ev: React.KeyboardEvent<HTMLElement>) => onKeyDown(itemKey!, ev)}
           aria-label={tab.ariaLabel}
-          role="tab"
+          role={tab.role||"tab"}
```

Comment:
This is the expected formatting which should have been applied on precommit:
```suggestion
          role={tab.role || 'tab'}
```